src/js/button.js

```diff
@@ -3,10 +3,7 @@
  */
 import ClickableComponent from './clickable-component.js';
 import Component from './component';
-import * as Events from './utils/events.js';
```

Comment:
There are removals like this scattered around: they were unused modules.

_Note_: if we're only trying to import the module, we should be doing something like `import './utils/events'`.